app/src/main/res/values/strings.xml

```diff
@@ -91,21 +91,24 @@
 
     <!-- App Backup and Restore State -->
 
+    <string name="backup_section_system">System Apps</string>
     <string name="backup_sms">SMS text messages</string>
     <string name="backup_settings">Device settings</string>
     <string name="backup_call_log">Call history</string>
     <string name="backup_contacts">Local contacts</string>
+    <string name="backup_section_user">User Apps</string>
     <!-- This text gets shown for apps that the OS did not try to backup for whatever reason e.g. no backup was run yet -->
     <string name="backup_app_not_yet_backed_up">Waiting to back up…</string>
     <string name="restore_app_not_yet_backed_up">Was not yet backed up</string>
     <string name="backup_app_was_stopped">Not backed up as it wasn\'t used recently</string>
     <string name="restore_app_was_stopped">Was not backed up as it hadn\'t been used recently</string>
     <string name="backup_app_no_data">App reported no data for backup</string>
-    <string name="backup_app_not_allowed">App doesn\'t allow backup</string>
+    <string name="backup_app_not_allowed">App doesn\'t allow backup</string> <!-- TODO remove -->
```

Comment:
Just curious, anything blocking now for removing this? And why remove it?

Reply:
Good catch! As the not allowed apps are now in their own section without repeating their status text, this string isn't needed anymore. I've now removed it.